src/Jira/Api.php

```diff
@@ -170,6 +170,21 @@ protected function clearLocalCaches()
 		$this->resolutions = null;
 	}
 
+    /**
+     * Helper method to formulate a query string from supplied parameter array.
+     *
+     * @param $params Array of URL parameters: [key => value]
+     * @return string
+     */
+	protected function formulateQueryString($params)
```

Comment:
There is PHP built-in function, that does it: http://php.net/manual/en/function.http-build-query.php